lib/declarations.d.ts

```diff
@@ -42,6 +42,8 @@ declare module Server {
 		solutionPath: string;
 		relativePath: string;
 		localFile?: string;
+		disposition: string;
```

Comment:
I'm not sure what is the implication if it is declared with or without `?`. Only older versions of the server will never return it. When I merge the PR in Ice it will always be there. How do you think it would be better - to leave it as required or not?